libs/components/layout/src/lib/modules/box/box-header.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="sky-box-header">
-  <span skyTrim><ng-content></ng-content></span
+  <span id="sky-sr-label-box" skyTrim><ng-content></ng-content></span
```

Comment:
Hardcoding an ID here will cause an ID clash when there is more than one box header component on a page and break accessibility. Since you reference this same ID in the box component, use [`SkyIdService`](https://developer.blackbaud.com/skyux/components/id#class-skyidservice) in that component to generate an ID and provide it to the box header component using an injection token so the box header can then bind this element's `[attr.id]` attribute to it and pass it on to the content info service.